libhif/CMakeLists.txt

```diff
@@ -126,7 +126,7 @@ if (GOBJECT_INTROSPECTION_FOUND)
         pkg_check_variable(gobject-introspection-1.0 g_ir_scanner)
         pkg_check_variable(gobject-introspection-1.0 g_ir_compiler)
 
-        set(GIR_PREFIX "Hif-${HIF_SO_VERSION}.0")
```

Comment:
I would prefer to set nsversion to `${HIF_SO_VERSION}.0` instead of changing filename. It's common practice to have it like this.